src/synology_dsm/api/download_station/__init__.py

```diff
@@ -32,17 +37,50 @@ def update(self):
                 self._tasks_by_id[task_data["id"]] = SynoDownloadTask(task_data)
 
     # Global
+    def update_info(self):
```

Comment:
what is the intention of this dedicated update method?
Those the user is forced to do an manually initial update before info can be accessed